teuthology/task/kernel.py

```diff
@@ -188,10 +188,21 @@ def need_to_install(ctx, role, version):
                                                                  want=version))
 
     if '.' in str(version):
-        # version is utsrelease, yay
         if cur_version == version:
             log.debug('utsrelease strings match, do not need to install')
             ret = False
+        our_os = teuthology.get_distro(ctx)
```

Comment:
maybe it makes sense to use `os_type` instead of `our_os`.

Reply:
done